### ISSUE TITLE
Update CONTRIBUTING.md Section 1.5.a

### ISSUE THREAD

**Issue by k-cardon** [hackforla/website#7642]
### Prerequisite
1. Be a member of Hack for LA. (There are no fees to join.) If you have not joined yet, please follow the steps on our [Getting Started page](https://www.hackforla.org/getting-started) and attend an onboarding session.
2. Before you claim or start working on an issue, please make sure you have read our [How to Contribute to Hack for LA Guide](https://github.com/hackforla/website/blob/7f0c132c96f71230b8935759e1f8711ccb340c0f/CONTRIBUTING.md).

### Overview
Some devs running WSL Ubuntu have been blocked by permission errors when running `docker-compose up`. This issue updates the contributing guide so that anyone who encounters this problem can quickly resolve it.

### Action Items
- [x] In your local IDE, navigate to `CONTRIBUTING.md`
- [x] **_UPDATED 4-20-25:_** Navigate to 1.5.a Docker installation troubleshooting and, at the end of that section, add:
```
If you are using Windows Subsystem for Linux (WSL) and finding a permission error when running `docker-compose up`, the issue might be caused by a version of Docker that relies on a buggy version of Go. In your terminal, run `docker version` to see which `Go version:` is listed. Any version less than `go1.20.0` has a problem and indicates that your Docker needs to be updated.
```
- [x] With CONTRIBUTING.md still open, make a note of the section in which the replaced content appears, so that you will know where to look in the document to preview the change.
- [x] Changes to CONTRIBUTING.md cannot be tested locally, rather they must be tested after pushing the issue branch to your fork of the repository.  Push your issue branch in the usual manner, but before creating the Pull Request, check your updates using this test URL.  Also store the test URL for use in a later step:
```
https://github.com/[REPLACE WITH GITHUB HANDLE]/website/blob/[REPLACE WITH NAME OF ISSUE BRANCH]/CONTRIBUTING.md
```
   (for example: https://github.com/bonniewolfe/website/blob/issue-branch-1234/CONTRIBUTING.md)
- [x] Create a pull request with your changes.  In the Pull Request, after the "Why did you make the changes" section, add this line to help reviewers, replacing the text in brackets (and the brackets) with the test URL from the previous Action Item.
```
For Reviewers: Do not review changes locally, rather, review changes at [REPLACE WITH TEST URL]
```

### Resources/Instructions
[CONTRIBUTING.md](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md)
[Getting started with writing and formatting on GitHub](https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github)

**Comment by HackforLABot** [comment]
Hello @aswutmaxcy, we appreciate you taking on issue #7642, however it looks like you've already worked on enough issues of this complexity. Try checking out some of the issues of the next complexity from the Prioritized Backlog :)

We are going to unassign you from this issue so you can take on another issue.

Hfla appreciates you! :)

P.S. There is one exception to this rule/automation, and that is if you were away for a long time, and need to do the issue ladder again. If that is the case, please post the following note on the issue and on your Skills Issue (Pre-work Checklist). A Merge team member will reassign you to this issue, and will help you get assigned to subsequent issues up to medium size.

```
I am returning after a significant time away, and need to do the issue ladder again. Please assign me back to this issue.
```

**Comment by HackforLABot** [comment]
Hello @mchait18, we appreciate you taking on this issue, however it looks like you're already working on another issue at this time. Please wait until your current issue is merged before taking on another issue. :)

We are going to unassign you from this issue so you can focus on your current issue.

Hfla appreciates you! :)

**Comment by HackforLABot** [comment]
Hi @jsnhn, thank you for taking up this issue! Hfla appreciates you :)

Do let fellow developers know about your:-
i. **Availability:** (When are you available to work on the issue/answer questions other programmers might have about your issue?)
ii. **ETA:** (When do you expect this issue to be completed?)

You're awesome!

P.S. - You may not take up another issue until this issue gets merged (or closed). Thanks again :)

**Comment by HackforLABot** [comment]
Hi @ryanfkeller, thank you for taking up this issue! Hfla appreciates you :)

Do let fellow developers know about your:-
i. **Availability:** (When are you available to work on the issue/answer questions other programmers might have about your issue?)
ii. **ETA:** (When do you expect this issue to be completed?)

You're awesome!

P.S. - You may not take up another issue until this issue gets merged (or closed). Thanks again :)

**Comment by ryanfkeller** [comment]
**Availability:** Weekdays 12-6p
**ETA Ready for Review:** 6/5/2025
